docs/core/introduction.md

```diff
@@ -179,231 +77,172 @@ A .NET app is built from source code by using [MSBuild](/visualstudio/msbuild/ms
 </Project>
 ```
 
-And here's one for a web app:
+Source code:
 
-```xml
-<Project Sdk="Microsoft.NET.Sdk.Web">
-  <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-  </PropertyGroup>
-</Project>
+```csharp
+Console.WriteLine("Hello, World!");
 ```
 
-In these examples, the `Sdk` attribute of the `Project` element specifies a set of MSBuild targets and tasks that build the project.  The `TargetFramework` element specifies the .NET version that the app depends on. You can edit the project file to add additional targets and tasks specific to the project.
-
-For more information, see [.NET project SDK overview](project-sdk/overview.md) and [Target frameworks](../standard/frameworks.md).
-
-### CI/CD
-
-MSBuild and the .NET CLI can be used with various continuous integration tools and environments, such as:
-
-* [GitHub Actions](https://github.com/features/actions)
-* [Azure DevOps](/azure/devops/user-guide/what-is-azure-devops)
-* [CAKE for C#](https://cakebuild.net/)
-* [FAKE for F#](https://fake.build/)
-
-For more information, see [Using .NET SDK and tools in Continuous Integration (CI)](tools/using-ci-with-cli.md)
+The app can be built and run with the [.NET CLI](tools/index.md):
 
-### NuGet
+```bash
+% dotnet run
+Hello, World!
+```
 
-[NuGet](/nuget/what-is-nuget) is an open-source package manager designed for .NET. A NuGet package is a *.zip* file with the `.nupkg` extension that contains compiled code (DLLs), other files related to that code, and a descriptive manifest that includes information like the package's version number. Developers with code to share create packages and publish them to [nuget.org](https://nuget.org) or a private host. Developers who want to use shared code add a package to their project and can then call the API exposed by the package in their project code.
+It can also be built and run as two separate steps. The following example is for an app that is named *app*:
 
-For more information, see [NuGet documentation](/nuget/).
+```bash
+% dotnet build
+% ./bin/Debug/net6.0/app 
+Hello, World!
+```
 
-### .NET Interactive
+## Binary distributions
 
-.NET Interactive is a group of CLI tools and APIs that enable users to create interactive experiences across the web, markdown, and notebooks.
+You can [download .NET](https://dotnet.microsoft.com/download/dotnet/6.0) for development or production scenarios:
```

Comment:
I would not use a specific version in the link, unless we are signing up for an annual reminder to update to the latest 'current'.

Reply:
There are a few 6.0 links that my commend equally applies too.

Reply:
```suggestion
You can [download .NET](https://dotnet.microsoft.com/download/dotnet) for development or production scenarios:
```